CHANGELOG

```diff
@@ -67,6 +67,8 @@ v 7.11.0 (unreleased)
   - Spin spinner icon next to "Checking for CI status..." on MR page.
   - Fix reference links in dashboard activity and ATOM feeds.
   - Ensure that the first added admin performs repository imports
+  - Update Asciidoctor gem to version 1.5.2. (Jakub Jirutka)
```

Comment:
Why do you think so? It’s an important change for users that uses AsciiDoc in GitLab.